src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsParameters.java

```diff
@@ -56,16 +57,23 @@
     private boolean wantClientAuth;
     private boolean needClientAuth;
 
+    /**
+     * Creates a HttpsParameters.
+     */
     protected HttpsParameters() {}
 
     /**
      * Returns the HttpsConfigurator for this HttpsParameters.
+     *
+     * @returns HttpsConfigurator for this instance of HttpsParameters.
```

Comment:
Probably would be worth saying if the result can be `null`. I guess it can. However I realize that this may be a more general issue with the whole API, so maybe what you have is OK for now, and we should leave further tightening of the spec to a followup.

Reply:
I have opened another issue for this -- https://bugs.openjdk.java.net/browse/JDK-8252822 -- and will address them there